test/src/vectorpostprocessors/TestFaceInfo.C

```diff
@@ -63,11 +63,6 @@ TestFaceInfo::execute()
     _face_area.push_back(p->faceArea());
     _elem_element_id.push_back(p->elem().id());
     _elem_element_side.push_back(p->elemSideID());
-    // the neighbor element might be a nullptr
```

Comment:
You guys should really use a Reporter for this object

Reply:
Good idea! For now, however I'd like to keep it as is. Isn't a VPP a Reporter underneath?

Reply:
Yeah - but with more context/other information. Looking at these diffs is miserable... looking at a json diff is wonderful